src/modules/user/user.service.ts

```diff
@@ -36,6 +36,41 @@ export class UserService {
 
 	private logger: CustomLogger;
 
+	async handleCreateGoogleUser({ email, name, sub: googleId }: GoogleUserInfo) {
+		try {
+			const userId = uuid();
+			const newUser: GoogleUser & TDynamoDBKeys = {
+				PK: `USER#${userId}`, // Partition key
+				SK: `#METADATA#${userId}`, // Sort key
+				createdAt: new Date().toISOString(),
+				email,
+				username: name, // User full name from google info
+				userId,
```

Comment:
no need for a separate declaration and better to rename it to `userUuid`, as you use uuid.

userUuid: uuid()